src/base/utils.js

```diff
@@ -85,6 +85,14 @@ export var Fullscreen = {
     } else if(document.msExitFullscreen) {
       document.msExitFullscreen()
     }
+  },
+  canGoFullscreen: function() {
```

Comment:
Yes that makes more sense I'll update it

> On 16 Jul 2016, at 00:38, Bruno Torres <notifications@github.com> wrote:
> 
> In src/base/utils.js:
> 
> > @@ -85,6 +85,14 @@ export var Fullscreen = {
> >      } else if(document.msExitFullscreen) {
> >        document.msExitFullscreen()
> >      }
> > +  },
> > +  canGoFullscreen: function() {
> I prefer using a standards based name, like fullscreenEnabled or isFullscreenEnabled. Why don't we go this way?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>